src/scripts/clipperUI/componentBase.ts

```diff
@@ -132,10 +126,36 @@ export abstract class ComponentBase<TState, TProps> {
 				element.style.outlineStyle = "none";
 			},
 			tabIndex: tabIndex,
-			"data-setnameforarrowkeynav": setNameForArrowKeyNav
+			"data-setnameforarrowkeynav": setNameForArrowKeyNav,
+			"data-defaultaction": defaultAction
 		};
 	}
 
+	private static enterKeySelection(e: KeyboardEvent) {
+		if (e.which === Constants.KeyCodes.enter) {
+			let defaultActionElement = document.querySelector("[data-defaultaction]:not([data-defaultaction='undefined'])");
```

Comment:
I was testing other web pages and noticed that sometimes, hitting the enter key on certain elements will behave the same hitting space. For example, in www.google.com, hitting enter on the About page triggers the About page, not the Search. Can you please doublecheck with Ailsa and Valentin on what the appropriate behavior should be?